xbmc/ServiceManager.cpp

```diff
@@ -23,9 +23,12 @@
 #include "utils/log.h"
 #include "interfaces/AnnouncementManager.h"
 #include "interfaces/generic/ScriptInvocationManager.h"
+#include "addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h"
 
 bool CServiceManager::Init1()
 {
+  V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```

Comment:
To prevent things like this was also my idea about `IAddonInterface`, but with the functions it have better a name as `CAddonInterfaceBase`, with the `I` on begin then really not right.